src/pcap/cmd/pcap-api/config.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"github.com/cloudfoundry/pcap-release/src/pcap/cmd"
+	"os"
+	"time"
+
+	"github.com/cloudfoundry/pcap-release/src/pcap"
+	"github.com/go-playground/validator/v10"
+	"gopkg.in/yaml.v3"
+)
+
+var DefaultAPIConfig = APIConfig{
+	CommonConfig: cmd.CommonConfig{
+		Listen: pcap.Listen{Port: 8080},
+		Buffer: pcap.BufferConf{
+			Size:       100,
+			UpperLimit: 95,
+			LowerLimit: 60,
+		},
+		LogLevel: "debug",
+		ID:       "test-api",
+	},
+	Agents: &pcap.AgentMTLS{
+		MTLS: nil,
+	},
+	DrainTimeout:       10 * time.Second,
+	ConcurrentCaptures: 5,
+	ManualEndpoints:    pcap.ManualEndpoints{Targets: []pcap.AgentEndpoint{{IP: "localhost", Port: 8083, Identifier: "test-agent/1"}}},
+}
+
+type APIConfig struct {
+	cmd.CommonConfig
+	Agents             *pcap.AgentMTLS `yaml:"agents"`
```

Comment:
Why is there an extra level that doesn't provide any additional fields?
```suggestion
	Agents             *pcap.MutualTLS `yaml:"agents"`
```

Reply:
The field name should probably be adjusted as well if this change is accepted

Reply:
Let us keep the logic and structure like it is. It might need to be adapted during the implementation of bosh resolver and finalizing of bosh case.